boot/boot_test.go

```diff
@@ -3341,7 +3341,7 @@ func (s *bootConfigSuite) TestBootConfigUpdateHappyNoKeysNoReseal(c *C) {
 	})
 	defer restore()
 
-	updated, err := boot.UpdateManagedBootConfigs(coreDev, s.gadgetSnap)
+	updated, err := boot.UpdateManagedBootConfigs(coreDev, s.gadgetSnap, "")
```

Comment:
same comment about having tests where cmdlineAppend is not empty

Reply:
Tests added